sample/src/main/java/io/ona/kujaku/sample/activities/CardActivity.java

```diff
@@ -0,0 +1,106 @@
+package io.ona.kujaku.sample.activities;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageButton;
+
+import com.mapbox.mapboxsdk.Mapbox;
+
+import org.json.JSONObject;
+
+import io.ona.kujaku.callbacks.AddPointCallback;
```

Comment:
Clean this import and other unused imports in the `CardActivity.java`

Reply:
Cleaned.